config/locales/pt.yml

```diff
@@ -191,13 +191,33 @@ pt:
   create_new_community: "Criar uma Nova Comunidade"
 
   # Mail messages
-  hello: "Olá %{user_name}"
-  new_user: "Obrigado pelo seu contacto. A sua mensagem foi recebida pela nossa equipa de suporte e iremos entrar em contacto consigo o mais rapidamente possível."
-  account_generated: "Foi criada uma nova conta para si. Para fazer login, use este link:"
-  above_this_line: "Escreva a sua resposta acima desta linha"
-  message_id: "Id da Mensagem"
-  message_response: "A seguinte mensagem foi criada em resposta à sua questão:"
-  view_online: "Ver isto online:"
+  hello: "Hello %{user_name}"
+  greetings: "Hello"
+  welcome: Welcome
+  new_user: "Thank you for contacting us.  Your message has been received by our support team and we will be in touch with you shortly with a response."
+  account_generated: "An account has been generated for you. To sign in, please use this link:"
+  above_this_line: "Make sure your reply appears above this line"
+  message_id: "Message ID"
+  message_response: "The following message has been posted in response to your question:"
+  view_online: "View this online:"
+
+  #Mail message Confirmation Instructions   #devise/mailer/confirmation_instructions.html.erb
+  confirmation_instructions:
+    account_email: "You can confirm your account email through the link below"
+    account_confirm: "Confirm my account"
+
+  #Mail Message Reset Password Instructions #devise/mailer/reset_password_instructions.html.erb
+  reset_password_instructions:
+      reset_link_request: "Someone has requested a link to change your password. You can do this through the link below"
+      change_password: "Change my password"
+      mail_ignore: "If you didn't request this, please ignore this email"
+      ignore_notice: "Your password won't change until you access the link above and create a new one"
+
+  #Mail Message Unlock Instructions #devise/mailer/unlock_instructions.html.erb
+  unlock_instructions:
+      account_locked: "Your account has been locked due to an excessive number of unsuccessful sign in attempts"
+      unlock_link: "Click the link below to unlock your account"
+      unlock_account: "Unlock my account"
 
```

Comment:
This is great, but these are already translated, take a look in `config/locales/devise`.   That being said, I am not sure the email template files are set up to use them.